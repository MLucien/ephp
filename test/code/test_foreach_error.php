<?php

foreach ($a as $b => $c)
	;

foreach ($x as $y)
	;

