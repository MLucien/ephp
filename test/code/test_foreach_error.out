
Notice: Undefined variable: a in {{CWD}}/test/code/test_foreach_error.php on line 3

Warning: Invalid argument supplied for foreach() in {{CWD}}/test/code/test_foreach_error.php on line 3

Notice: Undefined variable: x in {{CWD}}/test/code/test_foreach_error.php on line 6

Warning: Invalid argument supplied for foreach() in {{CWD}}/test/code/test_foreach_error.php on line 6
